docs/modules/ROOT/pages/documentation/microprofile/config/dynamo-db.adoc

```diff
@@ -0,0 +1,84 @@
+# DynamoDB Config Source
+
+The DynamoDB config source reads configuration properties from a NoSQL database table. 
```

Comment:
This statement is a bit vague. The feature doesn't allow to read properties from *any* NoSQL database, correct? You  must clarify that it only works for AWS DynamoDB. Also, it wouldn't hurt to add a short explanation on what DynamoDB is and how it works to help users understand the feature in a better manner